### PR TITLE
Some updates to the ksql workshop based on my trial run testing 5.0.x

### PR DESCRIPTION
**Pull request by rodesai**
- Include OSX in the note on the network naming
- Add some details on how to access kibana

### CODE REVIEW

File: ksql-workshop/ksql-workshop.adoc [hunk 1]
```diff
@@ -956,6 +956,15 @@ curl -X "POST" "http://localhost:18083/connectors/" \
...
 ----
 
+To access Kibana, first look up the address using docker ps:
+[source,bash]
+----
+$ docker ps
+71c9a39c5150        docker.elastic.co/kibana/kibana:6.3.0                                                   "/usr/local/bin/ki..."   11 minutes ago      Up 11 minutes       0.0.0.0:5601->5601/tcp                           kibana
+----
+
+Then, point your browser to that address (in this case, 0.0.0:5601)
```
Comment (ybyzek) [inline ksql-workshop/ksql-workshop.adoc]: Missing a `0`

File: ksql-workshop/ksql-workshop.adoc [hunk 2]
```diff
@@ -128,7 +128,7 @@ Note that the data is in Avro, so you will see lots of special characters in the
 
 Press Ctrl-C to cancel and return to the command prompt.
 
-NOTE: On Linux the name of the network is `ksqlworkshop_default` (no `-` between `ksql` and `workshop`).
+NOTE: On Linux and OSX the name of the network is `ksqlworkshop_default` (no `-` between `ksql` and `workshop`).
```
[the remark this answers is not in the document]
Comment (rodesai) [reply ksql-workshop/ksql-workshop.adoc]: You're right - I was doing some testing on linux as well and got confused.

[the remark this answers is not in the document]
Comment (rodesai) [reply ksql-workshop/ksql-workshop.adoc]:
```bash
Rohan-Desais-MBP15:ksql-workshop rohan$ docker network ls
NETWORK ID          NAME                   DRIVER              SCOPE
9eca3148112d        bridge                 bridge              local
d1e35dc6dbaf        ducknet                bridge              local
24d259000860        host                   host                local
c053e504b5b5        ksqlworkshop_default   bridge              local
20ff2964f748        none                   null                local
```

[the remark this answers is not in the document]
Comment (rodesai) [reply ksql-workshop/ksql-workshop.adoc]: What docker version do you run? I'm at 17.09

[the remark this answers is not in the document]
Comment (rmoff) [reply ksql-workshop/ksql-workshop.adoc]:
```
$ docker network ls
NETWORK ID          NAME                          DRIVER              SCOPE
199cc869324a        bridge                        bridge              local
d68dc3acb1bf        host                          host                local
a0c1883704c5        infoq-kafka-ksql_default      bridge              local
8979432d5abb        ksql-workshop_default         bridge              local
0298ac7ec41e        ksql-workshop_ksql-workshop   bridge              local
80239f841ac2        live-coding-ksql_default      bridge              local
8690b754eaab        none                          null                local

$ docker version
Client:
 Version:      18.06.0-ce-rc3
 API version:  1.38
 Go version:   go1.10.3
 Git commit:   cbfa3d9
 Built:        Thu Jul 12 05:15:46 2018
 OS/Arch:      darwin/amd64
 Experimental: true

Server:
 Engine:
  Version:      18.06.0-ce-rc3
  API version:  1.38 (minimum version 1.12)
  Go version:   go1.10.3
  Git commit:   cbfa3d9
  Built:        Thu Jul 12 05:26:13 2018
  OS/Arch:      linux/amd64
  Experimental: true
```

**Comment by ybyzek** [comment]
@rmoff @rodesai can y'all provide an update to this PR to refresh it or close the PR?

**Pull request closed without merging**